src/components/PageLayout/Navbar/Icons/NotificationIcon/NotificationIcon.vue

```diff
@@ -106,11 +106,13 @@
       // TODO https://github.com/eternagame/eternagame.org/issues/17 improve typing
       const res = response.data.data.entries as NotificationItem[];
 
-      this.notifications = res
+      if (res) {
```

Comment:
Could you briefly explain the rationale behind this change?

Reply:
While developing the page, I had encountered moments where `res` had been `null`, hence the if statement I'd placed. That being said, this happened much earlier in the dev process, so it could have been something I'd introduced but is no longer present. I just tested the page again and whatever issue had been present seems to be gone now. I'll revert this edit